localsetup.sh

```diff
@@ -1,12 +1,10 @@
 # pathvisio core
-mvn install:install-file -Dfile=localRepo/org.pathvisio.core.jar -DgroupId=org.pathvisio -DartifactId=pathvisio-core -Dversion=3.2.2 -Dpackaging=jar
+mvn install:install-file -Dfile=localRepo/org.pathvisio.core.jar -DgroupId=org.pathvisio -DartifactId=pathvisio-core -Dversion=3.3.0 -Dpackaging=jar
 
 # bridgedb 
-mvn install:install-file -Dfile=localRepo/org.bridgedb.jar -DgroupId=org.bridgedb -DartifactId=bridgedb -Dversion=2.0.0 -Dpackaging=jar
-mvn install:install-file -Dfile=localRepo/org.bridgedb.bio.jar -DgroupId=org.bridgedb -DartifactId=bridgedb-bio -Dversion=2.0.0 -Dpackaging=jar
+mvn install:install-file -Dfile=localRepo/org.bridgedb.jar -DgroupId=org.bridgedb -DartifactId=bridgedb -Dversion=3.0.13 -Dpackaging=jar
```

Comment:
I would suggest to remove this and the other `mvn install:install-file` lines for BridgeDb and let Maven figure it out with the regular Maven resolution.

Reply:
Yes, great suggestion for future version! But for now, I think this is the most stable way to get the app building

Reply:
@egonw : see also the documentation:

https://github.com/wikipathways/cytoscape-wikipathways-app/wiki/Add-local-dependency